packages/cli/core/src/commands/ui/deploy.ts

```diff
@@ -82,6 +84,13 @@ export default class Deploy extends CLICommand {
       default: 'coveo.deploy.json',
       required: false,
     }),
+    organization: Flags.string({
+      char: 'o',
+      helpValue: 'targetorganizationg7dg3gd',
+      required: false,
+      description:
+        'The unique identifier of the organization where to deploy the hosted page. If not specified, the organization you are connected to will be used.',
+    }),
```

Comment:
https://github.com/coveo/cli/blob/0809a3bbdb3774dffb4fdbacdf0a3beed9ba8924/packages/cli/core/src/lib/flags/snapshotCommonFlags.ts#L31-L37
Refactor this one maybe in commons or in another file then use it #673